server/database/queries/addUsers.js

```diff
@@ -0,0 +1,14 @@
+const dbconnection = require("../config/dbConnection");
+const bcrypt = require("bcryptjs");
+const addUser = (userinfo) => {
+  const { username, email, password } = userinfo;
+  // hash password to store it in  Database 
+  const hash = bcrypt.hashSync(password, 10);
+  const sql = {
+    text:
+      "INSERT INTO users(username,email,password,type) VALUES($1,$2,$3,$4)",
+    values: [username, email, hash, "subadmin"],
```

Comment:
instaed of hardcoding the value of type as `subadmin` you could make it in the database that the `default` value of `type` coulmn is `subadmin`. 
Change the sql file as follow :
` type VARCHAR(50) DEFAUL  'subadmin'`

And then you don't have to insert the value of `type` for the typical users, it will be automatically be set as `subadmin` for all new rows unless specified differently when inserting.

Reply:
Thank you , i did it  and edit on Database .